storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.4.rst

```diff
@@ -0,0 +1,59 @@
+============================
+ |Percona XtraBackup| 2.4.4
+============================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.4.4
+on July 25th 2016. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/>`_ and from :ref:`apt
+<apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the GA (Generally Available) stable release in the 2.4
+series.
+
+New features
+------------
+
+ |Percona XtraBackup| has been rebased on |MySQL| 5.7.13.
+
+Bugs Fixed
+----------
+
+ Due to a merge regression xtrabackup skipped the check for tablespace size in
+ the case of autoextend system tablespace, which could lead to an error when
+ backup was being prepared. Bug fixed :bug:`1550322`.
```

Comment:
xtrabackup reported the difference in the actual size of the system tablespace and the size which was stored in the tablespace header. This check now skipped for tablespaces with autoextend support.